docs/migrating/from-gatsby.md

```diff
@@ -0,0 +1,218 @@
+---
+description: Learn how to transition an existing Gatsby project to Next.js.
+---
+
+# Migrating from Gatsby
+
+This guide will help you understand how to transition from an existing Gatsby project to Next.js. Since both frameworks are built on top of React and share similar ideas, the transition can be broken down into a series of steps. First, let's talk about running the application.
```

Comment:
This is missing a promise statement / value proposition. Why do I want to migrate away from Gatsby? What does switching to Next.js give me?

Some things you can play with:

1. Choose SSG **or** SSR, instead of only SSG
1. API Routes
1. Faster development server
1. Incremental static (re-)generation

Reply:
Addressed 👍